docs/howto/deployment/wsgi/gunicorn.txt

```diff
@@ -21,16 +21,15 @@ Running Django in Gunicorn as a generic WSGI application
 ========================================================
 
 When Gunicorn is installed, a ``gunicorn`` command is available which starts
-the Gunicorn server process. At its simplest, gunicorn just needs to be called
-with the location of a module containing a WSGI application object named
-`application`. So for a typical Django project, invoking gunicorn would look
-like::
+the Gunicorn server process. At its simplest, gunicorn is called with the
+location of a module containing a WSGI application object named `application`.
+So for a typical Django project, invoking gunicorn would look like::
 
     gunicorn myproject.wsgi
 
 This will start one process running one thread listening on ``127.0.0.1:8000``.
-It requires that your project be on the Python path; the simplest way to ensure
-that is to run this command from the same directory as your ``manage.py`` file.
+It requires that your project be on the Python path; you can ensure this by
+running this command from the same directory as your ``manage.py`` file.
```

Comment:
I think something was lost here. This isn't the *only* way of ensuring the project is on the python path, it's just the simplest way. I prefer the old wording. Again "simple" here doesn't seem to be a comment on the reader's experience level